backend/LexData/Migrations/LexBoxDbContextModelSnapshot.cs

```diff
@@ -1164,6 +1164,75 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasForeignKey("LexCore.Entities.FlexProjectMetadata", "ProjectId")
                         .OnDelete(DeleteBehavior.Cascade)
                         .IsRequired();
+
+                    b.OwnsOne("LexCore.Entities.ProjectWritingSystems", "WritingSystems", b1 =>
+                        {
+                            b1.Property<Guid>("FlexProjectMetadataProjectId")
+                                .HasColumnType("uuid");
+
+                            b1.HasKey("FlexProjectMetadataProjectId");
+
+                            b1.ToTable("FlexProjectMetadata");
+
+                            b1.ToJson("WritingSystems");
+
+                            b1.WithOwner()
+                                .HasForeignKey("FlexProjectMetadataProjectId");
+
+                            b1.OwnsMany("LexCore.Entities.FLExWsId", "AnalysisWss", b2 =>
+                                {
+                                    b2.Property<Guid>("ProjectWritingSystemsFlexProjectMetadataProjectId")
+                                        .HasColumnType("uuid");
```

Comment:
Why is this here? What actual DB column does it correspond to? I have no idea why EF does this.